.github/workflows/build.yml

```diff
@@ -74,6 +74,8 @@ jobs:
           ref: ${{ env.HEAD_COMMIT }}
       - name: Set up Node
         uses: actions/setup-node@v1
+        with:
+          node-version: '14'
```

Comment:
Because I made the package node 14+, since that's what we develop in. (Our main `package.json`'s `volta.node` value is `14.17.0`.) Not having it is why the initial build failed. I'll admit that I went to bed before the second attempt finished - will go have a look now as to why it's still mad.